attention_and_transformers.ipynb

```diff
@@ -0,0 +1,4395 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.        # B,T,d_m -&gt; B, T, N, dm//N -&gt; B, N, T, dm//N
</pre><p>[Sugggestion] This is quite complicated to get at first, make sure to explain the motivation for this during the practice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>Very true, will make a note to tutors as well thanks. </p>